src/components/campaigns/DonationWishes.tsx

```diff
@@ -51,28 +62,49 @@ export default function DonationWishes({ campaignId, pageSize = 12 }: Props) {
           data.items &&
           data.items.map((wish) => (
             <Grid2
-              gap={2}
               container
               key={wish.id}
               direction="row"
-              sx={{ p: 2, bgcolor: 'grey.100', borderRadius: 5 }}>
+              sx={{ p: 2, bgcolor: 'grey.100', borderRadius: theme.spacing(2) }}>
               <Grid2 xs={12}>
                 <Stack direction="row" spacing={2}>
-                  <Stack>
-                    <AccountCircleIcon color="disabled" sx={{ fontSize: '2.3rem' }} />
-                  </Stack>
-                  <Stack direction="column" spacing={1}>
-                    <Stack direction="column">
-                      <Typography variant="caption" fontWeight={600}>
-                        {wish.person
-                          ? wish.person.firstName + ' ' + wish.person.lastName
-                          : t('campaigns:donations.anonymous')}
-                      </Typography>
-                      <Typography variant="caption" sx={{ color: 'gray' }}>
-                        {getExactDate(wish.createdAt, locale)}
-                      </Typography>
-                    </Stack>
-                    <Typography component="blockquote">{wish.message}</Typography>
+                  <Grid pt={0.7}>
+                    <AccountCircleIcon
+                      color="disabled"
+                      sx={{ fontSize: theme.typography.pxToRem(37) }}
+                    />
+                  </Grid>
+                  <Stack direction="column">
+                    <Typography
+                      variant="caption"
+                      sx={{
+                        fontWeight: 700,
+                        fontSize: theme.typography.pxToRem(16),
+                        color: '#343434',
```

Comment:
You can use one of the `theme.pallette.grey` colors instead of #343434

Reply:
They two greys we have in the theme are different and much lighter. We should probably introduce new colors to the theme. There are some common inlined color values.

Reply:
Might be a good idea to introduce a darker grey if the ones there are so light.
I thought the grey has values from 50-900, that are different :)